providers/asrockrack/asrockrack.go

```diff
@@ -0,0 +1,310 @@
+package asrockrack
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/bmc-toolbox/bmclib/internal/httpclient"
+	"github.com/bmc-toolbox/bmclib/providers"
+	"github.com/go-logr/logr"
+	"github.com/jacobweinstock/registrar"
+)
+
+const (
+	// BmcType defines the bmc model that is supported by this package
+	BmcType = "asrockrack"
+	// VendorID represents the id of the vendor across all packages
+	VendorID = "ASRockRack"
+	// ProviderName for the provider implementation
+	ProviderName = "asrockrack"
```

Comment:
looks like there might only be a need for one of these? `BmcType` or `ProviderName` or `VendorID`

Reply:
thanks, fixed in c0c4dfc295b